docs/guides/vpn/wireguard/client.md

```diff
@@ -29,6 +29,7 @@ For each new client, the following steps must be taken. For the sake of simplici
 
     echo "[Interface]" > "${name}.conf"
     echo "Address = $ipv4/32, $ipv6/128" >> "${name}.conf"
+    echo "DNS = 10.100.0.1" >> "${name}.conf" #Specifying DNS Server
```

Comment:
```suggestion
    echo "DNS = $serv4" >> "${name}.conf" #Specifying DNS Server
    echo "DNS = $serv6" >> "${name}.conf" #Specifying DNS Server
```

Reply:
I think we need to specify both the addresses (comma separated) on the same line, right?

Reply:
Both ways seem OK, based on `man wg-quick`

> DNS  —  a comma-separated list of IP (v4 or v6) addresses to be set as the interface's DNS servers, or non-IP hostnames to be set as the interface's DNS search domains. May be specified multiple times.